bootstrap/sql/org.postgresql.Driver/v009__create_db_connection_info.sql

```diff
@@ -220,3 +220,7 @@ WHERE serviceType = 'Salesforce';
 UPDATE dbservice_entity
 SET json = json::jsonb #- '{connection,config,supportsProfiler}'
 WHERE serviceType = 'DynamoDB';
+
+-- Update TagLabels source from 'Tag' to 'Classification' after #10486
+UPDATE table_entity SET json = REGEXP_REPLACE(json::text, '"source"\s*:\s*"Tag\"', '"source": "Classification"', 'g')::json;
```

Comment:
should we end with `::json` or `::jsonb`?

Reply:
Yes, yo are right. I will change it.